src/itemAction/index.tsx

```diff
@@ -29,6 +30,7 @@ class ItemAction extends PureComponent<ItemActionProps> {
 
   render() {
     const {
+      highlighted = true,
```

Comment:
We use something like `static defaultProps: Partial<**> = {}` for default values

Reply:
Done, thanks !